src/bci_build/package/versions.py

```diff
@@ -126,6 +126,7 @@ def get_pkg_version(pkg_name: str, os_version: OsVersion) -> str:
 #: projects from which to take package versions
 _OBS_PROJECTS: dict[OsVersion, str] = {
     **{OsVersion(ver): f"SUSE:SLE-15-SP{ver}:Update" for ver in range(3, 8)},
+    **{OsVersion.SLE16_0: "SUSE:SLFO:Main:Build"},
     **{OsVersion.TUMBLEWEED: "openSUSE:Factory"},
```

Comment:
Nit: these two could be one dictionary:
```suggestion
    **{OsVersion.SLE16_0: "SUSE:SLFO:Main:Build", OsVersion.TUMBLEWEED: "openSUSE:Factory"},
```

Reply:
resolved